PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -2000,7 +2008,7 @@ void ScintillaEditView::activateBuffer(BufferID buffer, bool force)
 	runMarkers(true, 0, true, false);
 
 	setCRLF();
-
+	execute(SCI_SETCHANGEHISTORY, SC_CHANGE_HISTORY_ENABLED | SC_CHANGE_HISTORY_MARKERS);

```

Comment:
@donho I don't think you want to enable it always here, I think you want to set it to the value of on/off depending on the UI:

```C++
	NppParameters& nppParam = NppParameters::getInstance();
	const ScintillaViewParams& svp = nppParam.getSVP();
	int enabledCH = svp._isChangeHistoryEnabled ? (SC_CHANGE_HISTORY_ENABLED | SC_CHANGE_HISTORY_MARKERS) : SC_CHANGE_HISTORY_DISABLED;
	execute(SCI_SETCHANGEHISTORY, enabledCH);

```